src/ResourceStorage/PRIVACY.md

```diff
@@ -0,0 +1,26 @@
+# ILIAS Resource Storage Service (IRSS) Privacy
+Disclaimer: This documentation does not warrant completeness or correctness. Please report any missing or wrong information using the [ILIAS issue tracker](https://mantis.ilias.de) or contribute a fix via [Pull Request](../../docs/development/contributing.md#pull-request-to-the-repositories).
+
+## General Information
+The IRSS is a special case as it is managing the storage of resources which are given to it and obtained by other Services and Modules. Therefore most of the privacy relevant parts take part in these services and components which use the IRSS. Currently these are the File-Object and the MainMenu, but in the future more Modules and Services will be migrated from the old storage to the IRSS.
+
```

Comment:
The IRSS is an abstraction layer for accessing and storing file-related data on top of the Filesystem. 
Services and components which use the IRSS might store, present, delete or export personal data. This is specified in their respective PRIVACY.md.